src/Sextant.Plugins.Popup.Tests/API/ApiApprovalTests.SextantPluginsPopup.netcoreapp2.2.approved.txt

```diff
@@ -0,0 +1,66 @@
+[assembly: System.Runtime.Versioning.TargetFramework(".NETStandard,Version=v2.0", FrameworkDisplayName="")]
+namespace Sextant.Plugins.Popup
```

Comment:
If you migrated to 3.1 you might not need this file anymore.

Reply:
fixed